contrib/opentimelineio_contrib/application_plugins/rv/example_otio_reader/otio_reader.py

```diff
@@ -231,13 +230,17 @@ def _create_item(it, track_kind=None):
         kind = "smptebars"
         if isinstance(it, otio.schema.Gap):
             kind = "blank"
-        new_media = ["{},start={},end={},fps={}.movieproc".format(
-            kind,
-            range_to_read.start_time.value,
-            range_to_read.end_time_inclusive().value,
-            range_to_read.duration.rate)]
+        new_media = [_create_movieproc(range_to_read, kind)]
+
+    try:
+        src = commands.addSourceVerbose(new_media)
+    except Exception as e:
+        # Perhaps the media was missing, if so, lets load an error
+        # source
+        print(e)
```

Comment:
just a naive RV question, does this need to be printed through some kind of error handler so that RV knows its an error?

Reply:
Ok, From what I can tell if I have "ERROR" in the string, the RV "Console" will raise. This seems to be the pattern in other python plugins that ship with RV. Updated code for that.